ephios/plugins/files/templates/files/document_list.html

```diff
@@ -0,0 +1,33 @@
+{% extends "core/settings/settings_base.html" %}
+{% load i18n %}
+
+{% block settings_content %}
+    <a class="btn btn-secondary" href="{% url "files:settings_document_create" %}"><span
+        class="fa fa-plus"></span> {% translate "Add file" %}</a>
+    <table class="table table-striped display mt-2">
+        <thead>
+            <tr>
+                <th>{% translate "Title" %}</th>
+                <th>{% translate "Action" %}</th>
+            </tr>
+        </thead>
+        <tbody>
+            {% for document in document_list %}
+                <tr>
+                    <td class="break-word">{{ document.title }}</td>
+                    <td class="d-flex">
+                        <a class="btn btn-secondary btn-sm text-nowrap me-1"
+                           href="{% url "files:document" document.pk %}"><span
+                            class="fa fa-download"></span> <span class="d-none d-md-inline">{% translate "Download" %}</span></a>
+                        <a class="btn btn-secondary btn-sm text-nowrap me-1"
+                           href="{% url "files:settings_document_edit" document.pk %}"><span
+                            class="fa fa-edit"></span> <span class="d-none d-md-inline">{% translate "Edit" %}</span></a>
+                        <a class="btn btn-secondary btn-sm text-nowrap me-1"
+                           href="{% url "files:settings_document_delete" document.pk %}"><span
+                            class="fa fa-trash"></span> <span class="d-none d-md-inline">{% translate "Delete" %}</span></a>
+                    </td>
+                </tr>
```

Comment:
We should really avoid introducing new html tables in templates. We should redesign this using bootstraps grid system and implement it for pages and qualifications as well, possibly including functionality for sorting/filtering like the datalist we once had. I'd be fine with moving this to an extra issue.

Reply:
yeah this is actually just copy pasta from somewhere else, I don't even remember.
I'll move it to a separate issue to get this merged soonish